nautobot/dcim/templates/dcim/cable_connect.html

```diff
@@ -33,15 +33,9 @@ <h3>{% block title %}Connect {{ termination_a.device }} {{ termination_a }} to {
                         {% if termination_a.device %}
```

Comment:
Would be good to manually test this template/form as I don't think we have good UT coverage of it.

Reply:
I will test it manually. But is this `CableConnectFormTestCase` for this template or am I mistaken?

Reply:
Sounds like it. :)